### PR TITLE
feat: [MJO-001] - Add Documentation on Repository

### PR DESCRIPTION
**Pull request by arthur404dev**
Add documentation for both document and reports

### CODE REVIEW

**Review by Wtheodoro — APPROVED** [review]
Very good my friend.

File: document.md [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a document
```
Comment (MiguelMachado-dev) [inline document.md:1]: Nice!

**Review by MiguelMachado-dev — APPROVED** [review]
LGTM

**Review by MBrunoS — APPROVED** [review]
LGTM

**Review by rodrigofmeneses — APPROVED** [review]

**Pull request merged**